Mathlib/Probability/Distributions/Exponential.lean

```diff
@@ -0,0 +1,283 @@
+/-
+Copyright (c) 2023 Claus Clausen. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Claus Clausen, Patrick Massot
+-/
+
+import Mathlib.Analysis.SpecialFunctions.Gaussian
+import Mathlib.Probability.Notation
+import Mathlib.Probability.Cdf
+
+/-! # Exponential distributions over ℝ
+
+Define the Exponential Measure over the Reals
+
+## Main definitions
+* `exponentialPdfReal`: the function `r x ↦ r * (Real.exp (-(r * ↑x))` for `0 ≤ x`
+  or `0` else, which is the probability density function of a exponential distribution with
+  rate `r` (when `hr : 0 < r`).
+* `exponentialPdf`: `ℝ≥0∞`-valued pdf,
+  `exponentialPdf r = ENNReal.ofReal (exponentialPdfReal r)`.
+* `expMeasure`: an exponential measure on `ℝ`, parametrized by its rate `r`.
+* `exponentialCdfReal`: the Cdf given by the Definition of CDF in `ProbabilityTheory.Cdf` on
+
+## Main results
+* `ExpCdf_eq`: Proof that the `exponentialCdfReal` given by the Definition equals the known
+  function given as `r x ↦ 1 - (Real.exp (-(r * ↑x))` for `0 ≤ x` or `0` else.
+-/
+
+open scoped ENNReal NNReal Real
+
+open MeasureTheory Real Set Filter Topology
+
+@[simp]
+lemma compl_setOf_le (y : ℝ) : {x : ℝ | y ≤ x}ᶜ =  {x | x < y} := by
+  ext x;
+  constructor <;>
+  simp only [ge_iff_le, mem_compl_iff, mem_setOf_eq, not_le, imp_self]
+
+  /-- A Lebesgue Integral from -∞ to y can be expressed
+    as the sum of one from -∞ to 0 and 0 to x -/
+lemma lintegral_split_bounded {y z : ℝ}(f : ℝ → ENNReal) ( hzy : z ≤ y) :
+    ∫⁻ (x : ℝ) in Iic y, f x  =  (∫⁻ (x : ℝ) in Iio z, f x) +  ∫⁻ (x : ℝ) in Icc z y, f x := by
```

Comment:
Most often, you don't need to specify the type of the variable over which you integrate.
```suggestion
    ∫⁻ x in Iic y, f x  = ∫⁻ x in Iio z, f x + ∫⁻ x in Icc z y, f x := by
```
Also I think these parentheses are not needed.

Reply:
Did that, but I think the parantheses are needed if the integral is written without its measure, at least that gives me an error